styles.css

```diff
@@ -42,7 +42,7 @@
   }
 
   .my-logo {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
```

Comment:
- [ ] I see 😃that there is a problem with the contrast color here at the logo, so kindly try to fix this contrast where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| The logo |
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216157482-36d562de-5642-4708-9826-d30cf207dfd4.png)  |

Reply:
> Correction

|My Logo|Test contrast color|
|:---------:|:-------------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/216194352-42a0e304-6371-4525-aed6-c25e99ceb395.png)|![imagen](https://user-images.githubusercontent.com/90431831/216194764-af0220f6-f4b9-47da-bdb2-7de79cb0ce98.png)|